src/codegen/cppcg.cpp

```diff
@@ -1096,7 +1099,76 @@ wxString CppCodeGenerator::GetCode( PObjectBase obj, wxString name )
 	return code;
 }
 
-void CppCodeGenerator::GenClassDeclaration( PObjectBase class_obj, bool use_enum, const wxString& classDecoration, const EventVector &events )
+wxString CppCodeGenerator::GetDeclaration(PObjectBase obj, ArrayMap& arrays) {
+	wxString code;
+
+	// Get the name
+	auto propName = obj->GetProperty(wxT("name"));
+	if (propName)
```

Comment:
Use an early return to avoid the indentation:

```cpp
if (!propName) {
		// Object has no name, just get its code
		return GetCode(obj, wxT("declaration"));
}
```